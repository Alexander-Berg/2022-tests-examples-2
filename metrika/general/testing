BISHOP_DB_CONNMAXAGE=600
BISHOP_DB_CONNTIMEOUT=2
BISHOP_DB_HOST=vla-2xbvzkfutgmvxo3i.db.yandex.net,man-szpxal2d5jw0n1ls.db.yandex.net,sas-17qy960sguan4y80.db.yandex.net
BISHOP_DB_NAME=bishop_test
BISHOP_DB_PORT=6432
BISHOP_DB_USER=metrika_infrastructure
BISHOP_DJANGO_DEBUG=false
BISHOP_GUNICORN_BIND=[::]:80
BISHOP_GUNICORN_DISABLE_REDIRECT_ACCESS_TO_SYSLOG=true
BISHOP_GUNICORN_ERRORLOG=-
BISHOP_GUNICORN_GRACEFUL_TIMEOUT=120
BISHOP_GUNICORN_LOGLEVEL=INFO
BISHOP_GUNICORN_MAX_REQUESTS=50
BISHOP_GUNICORN_MAX_REQUESTS_JITTER=50
BISHOP_GUNICORN_SYSLOG_ADDR=unix:///dev/log#dgram
BISHOP_GUNICORN_SYSLOG_PREFIX=log
BISHOP_GUNICORN_SYSLOG=true
BISHOP_GUNICORN_TIMEOUT=120
BISHOP_GUNICORN_WORKERS_COUNT=6
BISHOP_IDM_CERT_ISSUER=CN=YandexInternalCA,DC=ld,DC=yandex,DC=ru
BISHOP_IDM_CERT_SUBJECT=emailAddress=pki@yandex-team.ru,CN=idm.test.yandex-team.ru,OU=ITO,O=Yandex LLC,L=Moscow,ST=Moscow,C=RU
BISHOP_IDM_ENABLED=true
BISHOP_IDM_INSTANCE=testing
BISHOP_IDM_URL=https://idm.test.yandex-team.ru
BISHOP_LOCK_BASE_PATH=/bishop
BISHOP_LOCK_FAKE=false
BISHOP_LOCK_ZAPI_ENV=testing
BISHOP_LOCK_ZAPI_GROUP=common
BISHOP_LOGGING_FILE=/ephemeral/bishop.log
BISHOP_LOGGING_STDOUT=false
BISHOP_LOGGING_SYSLOG=true
BISHOP_NOODLE_ADDRESS=https://noodle-test.mtrs.yandex-team.ru
BISHOP_TVM_CLIENT_ID=2001652
BISHOP_YANDEX_AUTH_TEST=false
BISHOP_YANDEX_AUTH_USER=volozh
BISHOP_ZOOKEEPER_HOST=mtzoo01et.metrika.yandex.net,mtzoo01ft.metrika.yandex.net,mtzoo01it.metrika.yandex.net
BISHOP_ZOOKEEPER_PORT=2181
