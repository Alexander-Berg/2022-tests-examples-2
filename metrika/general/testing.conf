[nginx]
/. = sec-01cn1nhcfem1z9wbx876pzas27[*.key]
/.:0444 = sec-01cn1nhcfem1z9wbx876pzas27[*.crt]

[statbox]
/push-client.yaml = push-client-testing.yaml:sec-01fje9xrpvd6wv9pbekdytpg8m
/push-client.yaml.tvm = sec-01fje9x6eb68aq3zg27sv0zm0p[client_secret]
