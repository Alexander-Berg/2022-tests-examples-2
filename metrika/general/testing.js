const csp = require('csp-presets-pack');

module.exports = {
    backend: {
        host: 'https://ofd.tst.yandex.ru'
    },

    blackbox: {
        api: 'blackbox-mimino.yandex.net'
    },

    bunker: {
        api: 'http://bunker-api-dot.yandex.net/v1',
        version: 'latest'
    },

    csp: {
        presets: {
            avatars: csp.avatars()
        }
    }
};
