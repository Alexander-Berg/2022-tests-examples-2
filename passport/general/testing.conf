[tvm_keyring]
/passport_api = tvm_keyring/testing.tpl:sec-01cj4dhv5ytg0agc54db6d3bb6[testing]

[statbox_push_client]
/push-client-backend.yaml.tvm = sec-01cj4dhv5ytg0agc54db6d3bb6[testing]

[backend_secrets]
/settings.py = python_settings.tpl:sec-01cjdcda3vy0q5vg05pk59cycf->passport_settings,sec-01ezsb0q3jsh6tcm0tqrh50gqk->dbscripts_settings

[historydb]
/. = sec-01d0xvxrrab3z13fhc2rdmnbbh[*.key]
