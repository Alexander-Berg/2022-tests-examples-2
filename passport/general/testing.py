from .default import *

from .secrets import (
    DB_USER,
    DB_SLAVE_USER,
    DB_PASSWORD,
    DB_SLAVE_PASSWORD,
)

CONDUCTOR_API_NO_CACHE = {
    'address': 'https://c.test.yandex-team.ru/api/',
    'timeout': 60,
    'retry': 2,
}

CONDUCTOR_API = {
    'address': 'https://c.test.yandex-team.ru/api-cached/',
    'timeout': 60,
    'retry': 2,
}

DATABASES = {
    'default': {
        'ENGINE': 'django.db.backends.mysql',
        'HOST': 'cnt-dbm-test.passport.yandex.net',
        'NAME': DB_NAME,
        'USER': DB_USER,
        'PASSWORD': DB_PASSWORD,
        'TEST_CHARSET': 'UTF8',
    },
    'slave': {
        'ENGINE': 'django.db.backends.mysql',
        'HOST': 'cnt-dbm-test.passport.yandex.net',
        'NAME': DB_NAME,
        'USER': DB_SLAVE_USER,
        'PASSWORD': DB_SLAVE_PASSWORD,
        'TEST_CHARSET': 'UTF8',
    },
}

DATABASE_SLAVES = ['slave']

GRANTS_MAIL_ENV_NAME = True

EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'

EXPORT_GRANTS_URL = 'https://grantushka-test.yandex-team.ru/grants/operations/'
