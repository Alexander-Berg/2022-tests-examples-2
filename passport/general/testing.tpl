{
  "client_id": 2000079,
  "client_secret": "{{ testing }}"
}
