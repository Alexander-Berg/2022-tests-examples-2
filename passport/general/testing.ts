import common from './common';

import type { Config } from './types';

const config: Config = {
    ...common,
    environment: 'testing',
    metrika: {
        id: '69331741',
    },
    blackbox: {
        host: 'blackbox-test.yandex.net',
    },
    passport: {
        baseUrl: 'https://passport-test.yandex.ru',
    },
    passportAPI: {
        ...common.passportAPI,
        baseUrl: 'http://127.0.0.1:6000',
        host: 'passport-test-internal.yandex.ru',
    },
    socialAPI: {
        ...common.socialAPI,
        baseUrl: 'https://api.social-test.yandex.ru',
        frontendUrl: 'https://social-test.yandex.ru',
        host: 'api.social-test.yandex.ru',
    },
    documentsAPI: {
        ...common.documentsAPI,
        baseUrl: 'https://documents-test.pers.yandex.net',
        host: 'documents-test.pers.yandex.net',
    },
    avatar: {
        ...common.avatar,
        url: common.avatar.url.replace('mds', 'mdst'),
    },
    cloud: {
        ...common.cloud,
        baseUrl: 'https://cloud-api.dst.yandex.net:8443',
    },
    geo: {
        ...common.geo,
        baseUrl: 'http://addrs-testing.search.yandex.net/search/stable',
    },
    csp: {
        ...common.csp,
        'img-src': (common.csp as ANY)['img-src'].map((rule: string) => rule.replace('mds', 'mdst')),
    },
    oauthUrl: 'https://oauth-test.yandex.ru',
};

export default config;
