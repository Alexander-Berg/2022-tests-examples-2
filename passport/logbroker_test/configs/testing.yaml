emitter:
  args:
    client_id: passport/lbc-logbroker-test
    data_port: 2135
    credentials_config:
      credentials_type: tvm
      tvm_alias: logbroker_api
    per_host_config:
      '^passport-test-f1.passport.yandex.net$':
        targets:
          - host: man.logbroker.yandex.net
            ssl: true
            topic: passport/logbroker-test-testing
          - host: myt.logbroker.yandex.net
            ssl: true
            topic: passport/logbroker-test-testing
      '^passport-test-s1.passport.yandex.net$':
        targets:
          - host: sas.logbroker.yandex.net
            ssl: true
            topic: passport/logbroker-test-testing
          - host: vla.logbroker.yandex.net
            ssl: true
            topic: passport/logbroker-test-testing
      '^passport-test-v1.passport.yandex.net$':
        targets:
          - host: iva.logbroker.yandex.net
            ssl: true
            topic: passport/logbroker-test-testing
    workers_count: 10
  class: passport.backend.logbroker_client.core.consumers.simple.native_emitter.PartitionsCountNativeEmitter
  storage:
    class: passport.backend.logbroker_client.core.runner.storages.memory.MemoryTaskStorage
blackbox:
  url: https://pass-test.yandex.ru/
  timeout: 1
  retries: 2
  use_tvm: false
  get_hidden_aliases: false
passport:
  url: https://passport-test-internal.yandex.ru
  timeout: 3
  retries: 2
  use_tvm: true
logpath: /var/log/yandex/logbroker-client/logbroker-test/
workers:
  args:
    handler:
      args:
        log_metrics: False
        push_metrics_to_xunistater: True
        message_class: passport.backend.core.protobuf.logbroker_test.logbroker_test_pb2.LogbrokerTestMessage
      class: passport.backend.logbroker_client.logbroker_test.handler.LogbrokerTestHandler
  class: passport.backend.logbroker_client.core.consumers.simple.native_worker.NativeLogbrokerWorker
  count: 10
