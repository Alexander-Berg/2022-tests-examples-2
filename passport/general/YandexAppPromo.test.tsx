import React from 'react';

import { YandexAppPromo } from '.';
import { createClientRender } from '../../libs/testing';

describe('YandexAppPromo', () => {
  const render = createClientRender();

  test('baseline snapshot', () => {
    const result = render(
      <YandexAppPromo
        lang="ru"
        title="Блабла"
        installMessage="asdasd"
        description="dasdasd"
        usrImg=""
      />,
    );

    expect(result.asFragment()).toMatchSnapshot();
  });
});
