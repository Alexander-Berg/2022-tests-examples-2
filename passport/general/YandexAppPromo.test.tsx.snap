// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YandexAppPromo baseline snapshot 1`] = `
<DocumentFragment>
  <div
    class="YandexAppPromo"
  >
    <div
      class="PhonePromo PhonePromo_full YandexAppPromo-Logo"
    />
    <div
      class="YandexAppPromo-Info"
    >
      <div
        class="YandexAppPromo-Message YandexAppPromo-Message_upper"
      >
        Блабла
      </div>
      <div
        class="YandexAppPromo-Message YandexAppPromo-Message_lower"
      >
        dasdasd
      </div>
      <div
        class="YandexAppPromo-Install"
      >
        <span
          class="YandexAppPromo-InstallText"
        >
          asdasd
        </span>
        <div
          class="YandexAppPromo-InstallButtons"
        >
          <div
            class="YandexAppPromo-InstallAndroid"
          >
            <a
              aria-label="Google Play"
              href="https://redirect.appmetrica.yandex.com/serve/458404701881276453?c=passport_bage"
            >
              <div
                class="GooglePlay GooglePlay_lang_ru"
              />
            </a>
          </div>
          <div
            class="YandexAppPromo-InstallIos"
          >
            <a
              aria-label="App Store"
              href="https://redirect.appmetrica.yandex.com/serve/386347106450828605?c=passport_bage"
            >
              <div
                class="AppStore AppStore_lang_ru"
              />
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
